RELEASE-NOTES.md

```diff
@@ -21,7 +21,7 @@
 ## New additions
 
 ## Fixes and improvements
-
+* Fixed default argument shown in `snow connection add`
```

Comment:
Can you move it to 3.2.0  to avoid unnecessary next PRs, because I'll cherrypick it to 3.2.0 :)

Reply:
Done